spring-batch-docs/asciidoc/step.adoc

```diff
@@ -682,19 +684,19 @@ should be configured as follows:
 @Bean
 public Step step1() {
 	return this.stepBuilderFactory.get("step1")
-				.faultTolerant()
 				.<String, String>chunk(2)
-				.retryLimit(3)
 				.reader(itemReader())
 				.writer(itemWriter())
+				.faultTolerant()
+				.retryLimit(3)
 				.retry(DeadlockLoserDataAccessException.class)
 				.build();
 }
 ----
 
 The `Step` allows a limit for the number of times an individual item can be retried and a
 list of exceptions that are 'retryable'. More details on how retry works can be found in
-<<step.adoc#retryLogic, retry>>.
+<<retry.adoc#retry, retry>>.
```

Comment:
The "step.adoc#retryLogic" link was referring to the same section. The new link refers to the "retry" chapter where more details can be found.